pkg/cloud/aws/services/cloudformation/bootstrap.go

```diff
@@ -281,6 +285,45 @@ func cloudProviderNodeAwsPolicy() *iam.PolicyDocument {
 	}
 }
 
+func makePath(dir string, fileName string) string {
+	if strings.HasSuffix(dir, "/") {
```

Comment:
Can [path.Join](https://golang.org/pkg/path/#Join) be used here instead?

Reply:
Thank you! I tried searching for this API and clearly, I didn't look long enough :)